packages/itwin/ec3-widget/src/test/Templates.test.tsx

```diff
@@ -0,0 +1,265 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import React from "react";
+import "@testing-library/jest-dom";
+import { cleanup, render, screen, waitForElementToBeRemoved, waitFor } from "@testing-library/react";
+import Templates from "../components/Templates";
+import TemplateMenu from "../components/TemplateMenu";
+import * as moq from "typemoq";
+import { IModelApp, IModelConnection } from "@itwin/core-frontend";
+import { AuthorizationClient} from "@itwin/core-common";
+import { Guid } from "@itwin/core-bentley";
+import { ODataEntityResponse, ODataItem, ODataResponse, ReportingClient } from "@itwin/insights-client";
+import { useActiveIModelConnection } from "@itwin/appui-react";
+import type { ReportCollection } from "@itwin/insights-client";
+import faker from "@faker-js/faker";
+import {  LabeledSelect } from "@itwin/itwinui-react";
+import { setupServer } from "msw/node";
+import { rest } from "msw";
+import userEvent from "@testing-library/user-event";
+//import type { ODataResponse } from "./generated/api";
+
+afterAll(() => {

```

Comment:
Move all of the required setup into the `describe` function. Usually there should be very little code (mainly imports) outside of `describe` in test files.

Reply:
Most of the setups has been moved to describe function